lib/Cake/Console/Templates/default/actions/controller_actions.ctp

```diff
@@ -132,7 +132,7 @@
  */
 	public function <?php echo $admin; ?>delete($id = null) {
 		$this-><?php echo $currentModelName; ?>->id = $id;
-		if (!$this-><?php echo $currentModelName; ?>->exists()) {
+		if (!$this-><?php echo $currentModelName; ?>->exists($id)) {
```

Comment:
Do we need to set the property on line 134 still?

Reply:
I believe it is still needed for `delete()` call below.

Reply:
It could be moved down to it then maybe.

Reply:
Done.